test/python/test_push.py

```diff
@@ -0,0 +1,25 @@
+import os
+import subprocess
+import tempfile
+import utils
+
+import image
+
+
+def test_push_simple(registry1, registry2, storage_dir, tmpdir):
+    new_image = utils.new_image_name()
+    # replica_image = utils.new_image_name()
```

Comment:
rm replica related lines - let us add those in a separate test when we fix replica support

Reply:
okay